src/renderer/components/wallet/txs/table/TxsTable.tsx

```diff
@@ -51,12 +59,25 @@ export const TxsTable: React.FC<Props> = (props): JSX.Element => {
   )
 
   const renderAddressWithBreak = useCallback(
-    (address: Address, key: string) => (
-      <Styled.Text key={key}>
-        <AddressEllipsis address={address} chain={chain} network={network} />
-      </Styled.Text>
-    ),
-    [chain, network]
+    (address: Address, key: string) => {
+      const displayText = FP.pipe(
+        oWalletAddres,
+        O.map((walletAddress) =>
+          walletAddress === address ? (
```

Comment:
Just a double check: Are both addresses from same kind ("ui addresses" vs. "addresses")? Just to make sure we are comparing same kind of addresses. 

Pls check if it still works for ETH.

Reply:
yep. we compare the same addresses
![image](https://user-images.githubusercontent.com/67793194/110509607-56780500-8113-11eb-890e-c7f9a57e4afd.png)
![image](https://user-images.githubusercontent.com/67793194/110509637-5d9f1300-8113-11eb-98fc-7070f081dfc8.png)
![image](https://user-images.githubusercontent.com/67793194/110509671-67c11180-8113-11eb-89dd-6887bd6a245c.png)